src/services/ProcessorService.js

```diff
@@ -48,10 +90,12 @@ processScan.schema = {
       "mime-type": Joi.string().required(),
       payload: Joi.object()
         .keys({
-          submissionId: Joi.string().required(),
           url: Joi.string().required(),
+          cleanDestinationBucket: Joi.string().default(config.get('aws.DEFAULT_CLEAN_BUCKET')).valid(config.get('WHITELISTED_CLEAN_BUCKETS')),
+          quarantineDestinationBucket: Joi.string().default(config.get('aws.DEFAULT_QUARANTINE_BUCKET')),
           fileName: Joi.string().required(),
-          status: Joi.string().required(),
+          callbackUrl: Joi.string(),
+          callbackTopic: Joi.string().default(config.get('AVSCAN_TOPIC')),
```

Comment:
we should also whitelist the topics

Reply:
Not really, the result can be posted anywhere depending on how the caller needs to use it